srcpkgs/urlscan/template

```diff
@@ -11,7 +11,7 @@ license="GPL-2.0-or-later"
 homepage="https://github.com/firecat53/urlscan"
 changelog="https://github.com/firecat53/urlscan/releases"
 distfiles="https://github.com/firecat53/urlscan/archive/${version}.tar.gz"
```

Comment:
```suggestion
distfiles="${PYPI_SITE}/u/urlscan/urlscan-${version}.tar.gz"
```
This way, you can eliminate the `pre_build()` part below entirely. I made a [commit upstream](https://github.com/firecat53/urlscan/commit/4598452a09030165b2eaa238e1065bcb233b5437) so that the manpage is also included in the pypi tarball. This release is no-op otherwise.

Reply:
Thanks, committed your suggestions.